integrations/extensions/docs/elasticsearch-install-and-setup/python-document-ingestion/README.md

```diff
@@ -0,0 +1,191 @@
+
+```bash
+python3 -m venv .venv
+source .venv/bin/activate
+python3 -m pip install --upgrade pip
+```
+
+
+### Step 2: Create all the necessary environment variables
+
+The `ES_INDEX_NAME` and `ES_PIPELINE_NAME` variables can be whatever you would like to name your index and ingestion pipeline for use throughout this process/guide as references in various steps below.
+
+  ```bash
+  export ES_URL=https://<hostname:port>
+  export ES_USER=<username>
+  export ES_PASSWORD=<password>
+  export ES_CACERT=<path-to-your-cert>
+  export ES_INDEX_NAME=<name-of-index>
+  export ES_PIPELINE_NAME=<name-of-pipeline>
+  export DOCS_DIR=<full-path-to-your-directory-of-documents>
+  ``` 
+
+### Step 3: Create an ELSER ingest pipeline with an inference processor
+
+If you already have an existing pipeline in your Elasticsearch instance that uses an inference processor with ELSER against the "text" field, you can choose to reuse that and skip creation of a new pipeline.
+
+If not, then below is a sample request to create a pipeline that transforms the "text" field using the ELSER model and produces the terms along with weights as a sparse vector in the "ml" field at index time.
+
+You will be able to reference this pipeline in the next few steps as a part of indexing the documents of choice.
+
+Learn more about [inference-ingest-pipeline](https://www.elastic.co/guide/en/elasticsearch/reference/8.10/semantic-search-elser.html#inference-ingest-pipeline) from the tutorial 
+
+```bash
+curl -X PUT "${ES_URL}/_ingest/pipeline/${ES_PIPELINE_NAME}?pretty" -u "${ES_USER}:${ES_PASSWORD}" \
+-H "Content-Type: application/json" --cacert "${ES_CACERT}" -d'
+{
+  "processors": [
+    {
+      "inference": {
+        "model_id": ".elser_model_1",
+        "target_field": "ml",
+        "field_map": {
+          "text": "text_field"
+        },
+        "inference_config": {
+          "text_expansion": {
+            "results_field": "tokens"
+          }
+        }
+      }
+    }
+  ]
+}'
+```	 
+
+
+
+### Step 4: Create an index for storing tokens as rank features
+
+```bash
+curl -X PUT "${ES_URL}/${ES_INDEX_NAME}?pretty" -u "${ES_USER}:${ES_PASSWORD}" \
+-H "Content-Type: application/json" --cacert "${ES_CACERT}" -d'
+{
+  "settings":{
+  "index":{
+  "default_pipeline":"'"${ES_PIPELINE_NAME}"'"
+  }
+},
+  "mappings": {
+    "properties": {
+      "ml.tokens": {
+        "type": "rank_features"
+      },
+      "text": {
+        "type": "text"
+      }
+    }
+  }
+}'
+```
+
+NOTE: `rank_features` only works for ELSER v1 model. ELSER v2 requires a different type for the tokens. ELSER v2 has only been available since Elastic 8.11. Learn more about ELSER v2 from [here](https://www.elastic.co/guide/en/machine-learning/current/ml-nlp-elser.html)
```

Comment:
ELSER v2 just requires the `sparse_vector` type, and everything else can stay the same. 

I have updated the [web crawler guide to support both ELSER models](https://github.com/watson-developer-cloud/assistant-toolkit/blob/master/integrations/extensions/docs/elasticsearch-install-and-setup/how_to_use_web_crawler_in_elasticsearch.md#step-3-build-an-elser-ingest-pipeline-with-a-chunking-processor).  It would be nice to update it here as well. But I will leave it to you to either update it in this PR or in a follow-up PR.

Reply:
This is a good point.  ELSER v2 is coming to IBM Cloud very soon, and it is already available in watsonx Discovery on CP4D, so it would be good to include instructions for v2 here too.